recipes/paho-mqtt-c/all/conanfile.py

```diff
@@ -90,8 +90,8 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "eclipse-paho-mqtt-c"
-        self.cpp_info.names["cmake_find_package_multi"] = "eclipse-paho-mqtt-c"
+        self.cpp_info.names["cmake_find_package"] = "PahoMqttC"
+        self.cpp_info.names["cmake_find_package_multi"] = "PahoMqttC"
```

Comment:
The thing is that `paho-mqtt-c` does not install cmake modules itself. So we don't know what the correct name is.
Granted, `paho-mqtt-cpp` uses [`PahoMqttC`](https://github.com/eclipse/paho.mqtt.cpp/blob/master/cmake/PahoMqttCppConfig.cmake.in), but that's not official either.

There is no good solution ATM.
Conan should have a feature to rename these components for consumers.

Could you please add a comment/FIXME, saying that this name is used by `paho-mqtt-cpp`?

Reply:
Hmm?

https://github.com/eclipse/paho.mqtt.c/blame/e047e25d34d53b4b265649144a3cac3b01eee76c/src/CMakeLists.txt#L308

Reply:
@Croydon has better eyesight then me apparently. Thanks!

Reply:
Looks like CMakeLists.txt from paho-mqtt-cpp is looking for PahoMqttC
[https://github.com/eclipse/paho.mqtt.cpp/blob/a2e66455337a7e40277ca65ef2a3b045eb223c30/src/CMakeLists.txt#L26](https://github.com/eclipse/paho.mqtt.cpp/blob/a2e66455337a7e40277ca65ef2a3b045eb223c30/src/CMakeLists.txt#L26)

Reply:
It packs its own cmake module: https://github.com/eclipse/paho.mqtt.cpp/blob/a2e66455337a7e40277ca65ef2a3b045eb223c30/cmake/FindPahoMqttC.cmake

But that probably doesn't work with a static paho-mqtt-c library?

Reply:
I think you're seeing this problem: https://github.com/eclipse/paho.mqtt.cpp/issues/264

I think the correct solution is to fix their cmake script and use the exported targets of paho.mqtt.c.
If you write the patch on top of their current master, maybe they accept it.

Reply:
Looks like I'll have to take a look at the paho.mqtt.cpp issue. Thanks,